mono/mini/TestDriver.cs

```diff
@@ -51,6 +52,9 @@ public class TestDriver {
 				} else if (args [j] == "--exclude") {
 					exclude [args [j + 1]] = args [j + 1];
 					j += 2;
+				} else if (args [j] == "--exclude-test") {
+					exclude_test.Add (args [j + 1]);
```

Comment:
--exclude excludes categories of tests. --exclude-test excludes one specific test.

My use case was that I had one specific test which was crashing, thus making it impossible to run the tester in general. Excluding this test allowed me to use the tester while my debugging of the bad test was ongoing.

It might make sense to merge this with the --exclude, but I was trying not to change the behavior of an existing feature.